lib/bolt/transport/base.rb

```diff
@@ -46,13 +46,13 @@ def initialize
         @logger = Logging.logger[self]
       end
 
-      def with_events(target, callback)
+      def with_events(target, callback, action)
```

Comment:
Should this be optional, or default to 'action' as well?

Reply:
This is only used within Bolt, so we should always have a defined action.ag